wsdl_rs/b_2/src/lib.rs

```diff
@@ -36,7 +39,10 @@ impl Validate for TopicExpressionType {}
     prefix = "wsnt",
     namespace = "wsnt: http://docs.oasis-open.org/wsn/b-2"
 )]
-pub struct FilterType {}
+pub struct FilterType {
+    #[yaserde(prefix = "wsnt", rename = "TopicExpression")]
+    pub topic_expression: TopicExpressionType,
```

Comment:
In WSDL, it's a sequence of `any` types:

```
  <xsd:complexType name="FilterType">
    <xsd:sequence>
      <xsd:any minOccurs="0" maxOccurs="unbounded"/>
    </xsd:sequence>
  </xsd:complexType>
```

so it shouldn't be a required field. For a lack of better handling of `any` types, let's make the field optional to not break other use-cases:

```
pub topic_expression: Option<TopicExpressionType>,
```

Reply:
done